### ISSUE TITLE
Move the transformers into an independant bundle

### ISSUE THREAD

**Issue by shavounet** [cleverage/process-bundle#92]
## Description

Is it a good idea ?

Transformers are a big part of the process bundle, and could be independent with a easier access to them.

**Comment by xaviermarchegay** [comment]
@tonongregory what is your take on it? it seems to be annoying to manage?

**Comment by VincentChalnot** [comment]
For your information I already kind a did that on a personal project for test purposes and it makes sense but the it breaks backward compatibility completely because it involves moving all transformer in a new namespace.

For example I have a bundle that handles object mapping/conversion a bit like the Serializer but more advanced, Transformers are really helpful and I'm not using anything else from the ProcessBundle.
Ref : https://github.com/VincentChalnot/ConverterBundle/blob/v1.0-dev/reference.yaml

Hey @shavounet are you still using the ProcessBundle ?

**Comment by shavounet** [comment]
Yep, 100% a major BC break, but worth it in some usecase (aka _calling transformers without the full process bundle_).

Nope, it didn't gain traction here, and now we're switching away from PHP

**Comment by njoubert-cleverage** [comment]
Actually, this proposal doesn't seems worth it due to major BC Break that occured.
